pkg/api/admin/platformworkloadidentityroleset_convert.go

```diff
@@ -21,12 +20,17 @@ func (c platformWorkloadIdentityRoleSetConverter) ToExternal(s *api.PlatformWork
 		},
 	}
 
-	for i, r := range s.Properties.PlatformWorkloadIdentityRoles {
-		out.Properties.PlatformWorkloadIdentityRoles[i].OperatorName = r.OperatorName
-		out.Properties.PlatformWorkloadIdentityRoles[i].RoleDefinitionName = r.RoleDefinitionName
-		out.Properties.PlatformWorkloadIdentityRoles[i].RoleDefinitionID = r.RoleDefinitionID
-		out.Properties.PlatformWorkloadIdentityRoles[i].ServiceAccounts = make([]string, 0, len(r.ServiceAccounts))
-		out.Properties.PlatformWorkloadIdentityRoles[i].ServiceAccounts = append(out.Properties.PlatformWorkloadIdentityRoles[i].ServiceAccounts, r.ServiceAccounts...)
+	for _, r := range s.Properties.PlatformWorkloadIdentityRoles {
+		role := PlatformWorkloadIdentityRole{
+			OperatorName:       r.OperatorName,
```

Comment:
Darn that should have been for the line below.

Reply:
OperatorName:       r.OperatorName, the new line between these two pieces. It matches the following line style:

RoleDefinitionName: 
r.RoleDefinitionName,


I'm just not sure why it's different, but it could be totally correct that's why this is a question.

Reply:
There's no newline between `RoleDefinitionName:` and `r.RoleDefinitionName,`, and none of the struct fields are done that way. The GitHub UI sometimes inserts newlines and indents to get things to fit on the screen... maybe it confused you here by doing that?

Reply:
Yep it's the UI then. Thanks I didn't know it did that (thanks GH).